src/index.ts

```diff
@@ -327,6 +331,25 @@ export const createInstance = (config: Config): Instance => {
       uploadObjectAsJson: async (
         object: Record<string, unknown>
       ): Promise<string> => apiClient.uploadObject(object),
+
+      startUrlUploadJob: (urls: string[]): Promise<UrlToJobId[]> => {
+        if (urls.length > 100) {
+          throw new Error(invalidInputMessage);
+        }
+
+        return apiClient.startBulkUpload(urls);
+      },
+
+      checkBulkUploadJob: (jobIds: string[]): Promise<UploadJobStatus[]> => {
+        if (jobIds.length > 100) {
+          throw new Error(invalidInputMessage);
+        }
+        return apiClient.checkBulkUploadJob(jobIds);
+      },
+
+      checkUploadJob: (jobId: string): Promise<UploadJobStatus> => {
+        return apiClient.checkBulkUploadJob([jobId]).then((value) => value[0]);
```

Comment:
Don't use `.then` since it's promise chaining, instead use `await`

Also something here seems off, `checkBulkUploadJob` shouldn't return an array, but it looks like you're indexing into an array here, make sure to inspect return types, it should be an object.